src/entity/Question/api.ts

```
@@ -2,22 +2,40 @@ import { api } from "@shared/api";
 
+export const deleteQuestion = async (params: {quizId: string, questionId: string}) => {
+  const route = `/quizzes/${params.quizId}/questions/${params.questionId}`;
+
+  return await api.delete<Question>(route);
+};
+
+
 export const updateQuestion = async (params: QuestionCreate) => {
   const route = `/quizzes/${params.quiz_id}/questions/${params.questionId}`;
 
   delete params["quiz_id"];
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)